internal/testutils/jimmtest/fixture.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2024 Canonical.
+package jimmtest
```

Comment:
you know..

Reply:
I wonder if this isn't the linter always adding this copyright header like this.